integration/tests/cook/test_cli.py

```diff
@@ -1423,30 +1423,10 @@ def test_jobs_json_and_one_per_line_mutually_exclusive(self):
         self.assertEqual(2, cp.returncode, cp.stderr)
         self.assertIn('not allowed with argument', cli.decode(cp.stderr))
 
-    def test_piping_from_jobs_to_kill(self):
-        name = uuid.uuid4()
-        cp, uuids = cli.submit_stdin(['sleep 300'] * 3, self.cook_url, submit_flags=f'--name {name}')
-        self.assertEqual(0, cp.returncode, cp.stderr)
-        user = util.get_user(self.cook_url, uuids[0])
-        jobs_flags = f'--user {user} --name {name} --running --waiting'
-        cp, jobs = cli.jobs_json(self.cook_url, jobs_flags)
-        self.assertEqual(0, cp.returncode, cp.stderr)
-        self.assertEqual(3, len(jobs))
-        cs = f'{cli.command()} --url {self.cook_url}'
-        command = f'{cs} jobs {jobs_flags} -1 | {cs} kill'
-        self.logger.info(command)
-        cp = subprocess.run(command, shell=True)
-        self.assertEqual(0, cp.returncode, cp.stderr)
-        cp, jobs = cli.jobs_json(self.cook_url, jobs_flags)
-        self.assertEqual(0, cp.returncode, cp.stderr)
-        self.assertEqual(0, len(jobs))
-        cp, jobs = cli.jobs_json(self.cook_url, f'--user {user} --name {name} --failed')
-        self.assertEqual(0, cp.returncode, cp.stderr)
-        self.assertEqual(3, len(jobs))
-
-    def test_piping_from_jobs_to_show_and_wait(self):
+    def test_piping_from_jobs_to_kill_show_wait(self):
```

Comment:
When test run in parallel what stops jobs from other tests showing up in the results of these commands?

Reply:
All of the listing is done by `--name`, which is unique to this test